backend/app/services/export.py

```
@@ -0,0 +1,350 @@
+
+        await self.session.commit()
+        return export
+
+    async def generate_export(self, export_id: UUID) -> str:
+        """Generate the export file."""
+        export = await self.export_repo.get_by_id(export_id)
+        if not export:
+            raise ValueError("Export not found")
+
+        if export.status != ExportStatus.PENDING.value:
+            raise ValueError(f"Export is not in pending status: {export.status}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)